### PR TITLE
refine docs for MTNet and LSTM

### PR DESCRIPTION
**Pull request by TheaperDeng**
related to issue #2299
original PR: https://github.com/intel-analytics/analytics-zoo/pull/2345

### CODE REVIEW

**Comment by TheaperDeng** [comment]
PR validation pass: http://10.239.47.210:18888/job/ZOO-PR-Validation/3545/
APP test: http://10.239.47.210:18888/job/ZOO-PR-Pip-Apps-Integration-Part4/18/

**Comment by TheaperDeng** [comment]
http://10.239.47.210:18888/job/ZOO-PR-Validation/3582/

File: docs/docs/Zouwu/Algorithm/MTNetAlgorithm.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# MTNet Algorithm
+
+As we have mentioned before, MTNet is a memory-network based solution for multivariate time-series forecasting. The input data has a form of time series signal with several variables observed in each time stamp. Input data is divided into two parts, long-term historical data **{X_i}** and a short-term data series **Q**. Long-term historical data typically has several data series and each of them has the same length of the short-term data series.
```
Comment (shane-huang) [inline docs/docs/Zouwu/Algorithm/MTNetAlgorithm.md:3]: Remove "As we mentioned before"

File: docs/docs/Zouwu/Algorithm/MTNetAlgorithm.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# MTNet Algorithm
+
+As we have mentioned before, MTNet is a memory-network based solution for multivariate time-series forecasting. The input data has a form of time series signal with several variables observed in each time stamp. Input data is divided into two parts, long-term historical data **{X_i}** and a short-term data series **Q**. Long-term historical data typically has several data series and each of them has the same length of the short-term data series.
+
+One of the main modules of MTNet is its Encoder. It composes of three parts, a convolutional layer, an attention layer and a recurrent layer. For the convolutional layer, we have `cnn_hid_size` filters with width as the number of input feature `feature_dim` and height as `cnn_height`.  Then the convolution output is sent to a recurrent neural network with attention mechanism. You can have several RNN layers and the units number is set in `rnn_hid_size` The recurrent neural network is implemented as a GRU in this case.
```
Comment (shane-huang) [inline docs/docs/Zouwu/Algorithm/MTNetAlgorithm.md:5]: "composes of" -> "consists"

Comment (shane-huang) [reply docs/docs/Zouwu/Algorithm/MTNetAlgorithm.md:5]: >For the convolutional layer, we have `cnn_hid_size` filters with width as the number of input feature `feature_dim` and height as `cnn_height`. 

change to 

>The convolutional layer has `cnn_hid_size` filters, each of width `feature_dim` (i.e. the number of input features) and height `cnn_height`

File: docs/docs/Zouwu/Algorithm/MTNetAlgorithm.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+![MTnet_2](../../Image/ZouwuModel/MTnet_2.png)
+
+There is another auxiliary autoregressive model works independently. The autoregressive model assumes that the value to be forecasted is relevant to its previous value. `ar_window_size` states the number of previous value you want to use in the regression. The output feature number is also `target_dim`.
+
```
Comment (shane-huang) [inline docs/docs/Zouwu/Algorithm/MTNetAlgorithm.md:14]: >`ar_window_size` states the number of previous value you want to use in the regression. 

change to

>You can use `ar_window_size` to set the size of the history value you want lookback.

**Comment by TheaperDeng** [comment]
pr validation pass http://10.239.47.210:18888/job/ZOO-PR-Validation/3641/

**Pull request merged**